src/htr2hpc/train/data.py

```diff
@@ -19,7 +20,7 @@
 
 
 def get_segmentation_data(
```

Comment:
Since this is covering acquiring both training data for segmentation and for transcription tasks, the name `get_segmentation_data` might be confusing -- that and the name of the variable `segmentation_data` used later could perhaps be tidied up to make it clearer that it is general training_data or something. But not something to worry about until the code is more finalized.

Reply:
Ah, I already have a `get_training_data` method - and this does return a list of kraken `Segmentation` objects. I'm going to leave it for now, maybe if/when we refactor we can figure out better names.